services/chaindb/postgresql/blstoexecutionchanges.go

```diff
@@ -48,18 +48,21 @@ func (s *Service) setBLSToExecutionChanges(ctx context.Context, block *chaindb.B
 INSERT INTO t_block_bls_to_execution_changes(f_block_root
                                             ,f_block_number
                                             ,f_index
+											,f_validator_index
```

Comment:
It looks like there are tabs used here; whitespace in the commands should be spaces.  Please could you correct here and elsewhere this shows up?

Reply:
Fixed